lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1970,6 +1995,10 @@ bool CompilerInvocation::parseArgs(
     return true;
   }
 
+  if (ParseSymbolGraphArgs(SymbolGraphOpts, ParsedArgs, Diags, LangOpts)) {
+    return true;
+  }
```

Comment:
I understand this is probably the way it is for consistency reasons, but given that the function always returns `false`, I think it would be better to return `void` and avoid this branch, similar to `ParseModuleInterfaceArgs` earlier in this function.

Reply:
That makes total sense! I didn't realize that other parsing functions returned `void`. I'll update it.